acme/acme/standalone.py

```diff
@@ -176,10 +176,11 @@ def __init__(self, *args, **kwargs):
 class HTTP01Server(HTTPServer, ACMEServerMixin):
     """HTTP01 Server."""
 
-    def __init__(self, server_address, resources, ipv6=False):
+    def __init__(self, server_address, resources, ipv6=False, timeout=30):
+        # type: (Tuple[str, int], Set[Any], bool, float) -> None
```

Comment:
Would appreciate a hint on how I can get this mypy forward reference to `Set[HTTP01RequestHandler.HTTP01Resource]` instead of `Any` to work.

Reply:
Dod you try to import it? Mypy expects types to be imported like any other reference you would use in the actual code. For instance, `Any` is imported from `acme.magic_typing` that (conditionnally if available) delegates to the `typing` module.

Reply:
I did try it originally, yes. `tox -e mypy` doesn't like it:

>acme/acme/standalone.py:179: error: Name 'HTTP01RequestHandler.HTTP01Resource' is not defined

It seems like it's because it's a forward reference. If I move the entire `HTTP01Server` class below that of `HTTP01RequestHandler`, the error goes away ... is that what I should do?

Reply:
Ah yes, I understand. Indeed, you should !

Reply:
Thanks, you're always a great help!